session/bootstrap.go

```diff
@@ -607,6 +621,14 @@ func upgradeToVer20(s Session) {
 	doReentrantDDL(s, CreateStatsFeedbackTable)
 }
 
+func upgradeToVer21(s Session) {
+	mustExecute(s, CreateGCDeleteRangeDoneTable)
+
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range DROP INDEX element_id", ddl.ErrCantDropFieldOrKey)
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range DROP INDEX job_id", ddl.ErrCantDropFieldOrKey)
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range ADD UNIQUE INDEX delete_range_index (job_id, element_id)", ddl.ErrDupKeyName)
```

Comment:
Can we add index first?
If we drop the element id unique index, then two records inserted with the same element_id, then add unique index would fail.